Change in googleapis/gnostic breaks "make generate-openapi"

So https://github.com/googleapis/gnostic/commit/896953e6749863beec38e27029c804e88c3144b8 happened yesterday. Which is nice. Except it breaks `make generate-openapi`, and as a result, `make lint` and the `lint` context on all jx PRs:

```
Generating OpenAPI structs for Kubernetes Clients
./build/codegen --generator-version 36ebc4887cdc openapi --output-package=pkg/client --input-package=github.com/jenkins-x/**/pkg/apis --group-with-version=jenkins.io:v1
time="2020-02-01T17:14:56Z" level=info msg="installing k8s.io/kube-openapi/cmd/openapi-gen with version 36ebc4887cdc via 'go get' to /builder/home/.**/codegen/go" app=codegen
time="2020-02-01T17:15:34Z" level=info msg="generating Go code to /workspace/go/src/github.com/jenkins-x/** in package github.com/jenkins-x/**/pkg/client from package github.com/jenkins-x/**/pkg/apis\n" app=codegen
generating openapi structs to github.com/jenkins-x/**/pkg/client: generating open api dependency k8s.io/apimachinery:pkg/apis:meta:v1: error running go get -u -d k8s.io/apimachinery/..., output package k8s.io/apimachinery/pkg/util/strategicpatch
	imports github.com/googleapis/gnostic/OpenAPIv2: cannot find package "github.com/googleapis/gnostic/OpenAPIv2" in any of:
	/usr/local/go/src/github.com/googleapis/gnostic/OpenAPIv2 (from $GOROOT)
	/builder/home/.**/codegen/go/src/github.com/googleapis/gnostic/OpenAPIv2 (from $GOPATH): failed to run 'go get -u -d k8s.io/apimachinery/...' command in directory '', output: 'package k8s.io/apimachinery/pkg/util/strategicpatch
	imports github.com/googleapis/gnostic/OpenAPIv2: cannot find package "github.com/googleapis/gnostic/OpenAPIv2" in any of:
	/usr/local/go/src/github.com/googleapis/gnostic/OpenAPIv2 (from $GOROOT)
	/builder/home/.**/codegen/go/src/github.com/googleapis/gnostic/OpenAPIv2 (from $GOPATH)'
make: *** [Makefile.codegen:55: codegen-openapi] Error 1
```

As best as I can tell, this is because while `github.com/kubernetes/apimachinery` has a specific version of `github.com/googleapis/gnostic` in its `go.mod`, that's being ignored when we call `go get` within the codegen, so we're always getting `master`. And `master` is invalid for `k8s.io/apimachinery`'s dependency, since that depends on `github.com/googleapis/gnostic/OpenAPIv2`, but on `master`, there's `github.com/googleapis/gnostic/openapiv2`. So...that's a thing.

--- a/cmd/codegen/app/generate_openapi.go
+++ b/cmd/codegen/app/generate_openapi.go
@@ -79,12 +79,12 @@ func NewCmdCreateClientOpenAPI(genOpts GenerateOptions) *cobra.Command {
 	}
 
 	openAPIDependencies := []string{
-		"k8s.io/apimachinery:pkg/apis:meta:v1",
-		"k8s.io/apimachinery:pkg/api:resource:",
-		"k8s.io/apimachinery:pkg/util:intstr:",
-		"k8s.io/api::batch:v1",
-		"k8s.io/api::core:v1",
-		"k8s.io/api::rbac:v1",
+		"k8s.io/apimachinery?modules:pkg/apis:meta:v1",
+		"k8s.io/apimachinery?modules:pkg/api:resource:",
+		"k8s.io/apimachinery?modules:pkg/util:intstr:",
+		"k8s.io/api?modules::batch:v1",
+		"k8s.io/api?modules::core:v1",
+		"k8s.io/api?modules::rbac:v1",
 	}
 
 	moduleName := strings.TrimPrefix(strings.TrimPrefix(wd, filepath.Join(build.Default.GOPATH, "src")), "/")

